03-task-manager/starter/middleware/error-handler.js

```diff
@@ -0,0 +1,11 @@
+const { CustomAPIError } = require("../errors/custom-error")
+
+const errorHandlerMiddleware = (err, req, res, next) => {
+    if (err instanceof CustomAPIError) {
+        return res.status(err.statusCode).json({msg: err.message})
+    }
+    return res.status(500).json({message: 'There seems to be a problem, try again later'})
```

Comment:
Add the err variable to the JSON for returning what condition failed in the validation, for example, when the name is not informed to create a task.
You can do this like this: 
`return res.status(500).json({message: 'There seems to be a problem, try again later', err})`